src/shared/components/mutationTable/column/FunctionalImpactColumnFormatter.tsx

```diff
@@ -25,6 +25,7 @@ import GenomeNexusCache, {
 } from 'shared/cache/GenomeNexusCache';
 import _ from 'lodash';
 import { shouldShowMutationAssessor } from 'shared/lib/genomeNexusAnnotationSourcesUtils';
+import { selected } from 'pages/studyView/table/tables.module.scss';
```

Comment:
ideally shared components should not import from `pages`, but we probably broke that convention many times already, so not a big deal.

Reply:
Actrually this is an unused import. Removed this line.